### PR TITLE
Xbe fixes

### PR DESCRIPTION
**Pull request by smx-smx**
Fixed incorrect logic in XBE Loader

### CODE REVIEW

File: src/ImageLoaders/Xbe/XbeLoader.cs [hunk 1]
```diff
@@ -59,7 +59,7 @@ private XbeBuildType DetectBuildType()
                 return XbeBuildType.SegaChihiro;
             }
 
-            if ((ImageHeader.EntryPoint ^ XBE_XORKEY_ENTRYPOINT_RELEASE) > ImageHeader.BaseAddress)
+            if ((ImageHeader.EntryPoint ^ XBE_XORKEY_ENTRYPOINT_RELEASE) > 0x1000000)
```
Comment (uxmal) [inline src/ImageLoaders/Xbe/XbeLoader.cs:62]: Could you make the `0x1000000` a named constant, explaining its purpose? Remember the Reko convention is 
```C#
public const uint MagicConstant = 0x1000000
```

**Review by uxmal — COMMENTED** [review]
Thanks for the PR. One small nit and we're good to go.

File: src/ImageLoaders/Xbe/XbeLoader.cs (hunk 1 above)
Comment (smx-smx) [reply src/ImageLoaders/Xbe/XbeLoader.cs]: Done

**Comment by uxmal** [comment]
Thanks for the contribution!

**Pull request merged**